### ISSUE TITLE
Input cursor jumps to the end of text when it is modified

### ISSUE THREAD

**Issue by ardoramor** [kazupon/vue-validator#145]
I guess this is only me as no one else has posted this issue but I'm seeing the following behavior:

![validator-bug](https://cloud.githubusercontent.com/assets/6255652/12529234/9e457350-c17d-11e5-9145-cf5cf50ceae4.gif)

Template markup:

```
<validator name="onboardValidator">
    <form v-on:submit.prevent="onSubmit" novalidate>
        <fieldset>
            <legend>Settings</legend>
            <!-- ... -->
            <div class="form-group label-floating" :class="{'has-error': $onboardValidator.email.invalid}">
                <label class="control-label">E-mail</label>
                <input type="email" class="form-control" v-model="email" v-validate:email="[ 'required', 'email' ]"/>
                <p class="help-block">E-mail address associated with your account.</p>
            </div>
            <!-- ... -->
        </fieldset>
    </form>
</validator>
```

I am using:
- vue@1.0.15
- vue-validator@2.0.0-alpha.15
- bootstrap-material-design@0.5.7

The same was the case even in vue-validator@2.0.0-alpha.10.

This runs on Electron@0.35.4

The issue occurs on a regular input as well, event without regex match for `email`:

![validator-bug-regular](https://cloud.githubusercontent.com/assets/6255652/12529279/5140d986-c17e-11e5-9a30-63919e533eb9.gif)

Regular input markup:

```
<input type="text" v-model="email" v-validate:email="[ 'required' ]"/>
```

Basically, if I remove `v-model`, validator works fine but I need it to validate my model inputs. Any way I can help debugging this?

**Comment by kazupon** [comment]
related #141 #142 

Please use v2.0.0-alpha.16 :arrow_up: 
https://github.com/vuejs/vue-validator/releases/tag/v2.0.0-alpha.16

**Comment by ardoramor** [comment]
Yes, thanks! It works!
